metricbeat/module/kubernetes/util/kubernetes.go

```diff
@@ -760,8 +765,14 @@ func NewContainerMetadataEnricher(
 		return &nilEnricher{}
 	}
 	// We initialise the use_kubeadm variable based on modules KubeAdm base configuration
-	config.AddResourceMetadata.Namespace.SetBool("use_kubeadm", -1, commonMetaConfig.KubeAdm)
-	config.AddResourceMetadata.Node.SetBool("use_kubeadm", -1, commonMetaConfig.KubeAdm)
+	err = config.AddResourceMetadata.Namespace.SetBool("use_kubeadm", -1, commonMetaConfig.KubeAdm)
+	if err != nil {
```

Comment:
Although metricbeat is not related with this PR, I added checks here as well in order to have the same style in all beats